src/listeners/interactionCreate.ts

```diff
@@ -0,0 +1,25 @@
+import { Client, CommandInteraction, Interaction } from 'discord.js';
+import { Commands } from '../Commands';
+
+export default (client: Client): void => {
+  client.on('interactionCreate', async (interaction: Interaction) => {
+    if (interaction.isCommand()) {
+      await handleSlashCommand(client, interaction);
+    }
+  });
+};
```

Comment:
Nit: type is inferred automatically, `async` may be omitted
```suggestion
export default (client: Client): void => {
  client.on('interactionCreate', interaction => {
    if (interaction.isCommand()) {
      return handleSlashCommand(client, interaction);
    }
  });
};
```